package.json

```diff
@@ -61,7 +62,7 @@
     "rxjs": "^7.8.1",
     "server": "file:dist/server",
     "sqlite3": "^5.1.7",
-    "find-process": "^1.4.7"
+    "ws": "^8.18.0"
```

Comment:
used by one of the submodules and needs to be included in the top level package for cli to function